pkg/explorer/block.go

```diff
@@ -0,0 +1,29 @@
+package explorer
+
+import (
+	"fmt"
+	"github.com/tdex-network/tdex-daemon/pkg/httputil"
+	"net/http"
+	"strconv"
+)
+
+func (e *explorer) GetBlockHeight() (int, error) {
+	url := fmt.Sprintf(
+		"%v/blocks/tip/height",
+		e.apiUrl,
+	)
+	status, resp, err := httputil.NewHTTPRequest("GET", url, "", nil)
+	if err != nil {
+		return 0, err
```

Comment:
I would return -1 in case of error
```suggestion
		return -1, err
```